src/sage/knots/link.py

```diff
@@ -2362,6 +2403,48 @@ def regions(self):
             regions.append(region)
         return regions
 
+    def remove_loops(self):
+        r"""
+        Return an ambient isotopic link in which all loops are removed.
+
+        EXAMPLES::
+
+          sage: b = BraidGroup(4)((3, 2, -1, -1))
```

Comment:
```suggestion
        EXAMPLES::

            sage: b = BraidGroup(4)((3, 2, -1, -1))
```
The spacing of this block needs to be fixed.